modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -7,6 +7,8 @@
 
 You must build a container disk with a virtual machine image and push it to a container registry before it can used with a virtual machine. You can then either import the container disk into a PVC using a data volume and attach it to a virtual machine, or you can attach the container disk directly to a virtual machine as an ephemeral `containerDisk` volume.
 
+The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay documentation] for information about the default layer size of the registry and how you can change this value by editing the appropriate field in the YAML configuration file that is created when Red Hat Quay is first deployed.
```

Comment:
I like that the first sentence is now part of the body text (it's important!) but I think that the RH Quay part of this could be a note in itself. The Quay sentence is also quite long while also not providing specific information, so I'd cut it down. What about something like this?

```
The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. 
+
[NOTE]
====
For link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay], you can change the maximum layer size by editing the YAML configuration file that is created when Red Hat Quay is first deployed.
====
```

Thoughts?

Reply:
Works for me.